config/initializers/_config.rb

```diff
@@ -8,7 +8,7 @@ module Convection
   OpenStruct.new(
     access_token: ENV['ACCESS_TOKEN'] || 'replace-me',
     admin_names: (ENV['ADMIN_NAMES'] || 'Alice Betty Cindy').split,
-    admin_email_address: ENV['ADMIN_EMAIL_ADDRESS'] || 'consign@artsy.net',
+    admin_email_address: ENV['ADMIN_EMAIL_ADDRESS'] || 'sell@artsy.net',
```

Comment:
Is the env var also updated?
And if so, we also need to confirm it doesn't break stuffs elsewhere.